### PR TITLE
Adjust float format of ReleaseStat to avoid Excel Problem

### PR DESCRIPTION
**Pull request by fso42**
Having only 2 decimals leads to Excel changing these fields to dates.... I know...

- to avoid any confusion, it is set to 4 decimals now
- clean getInput.py with black and remove unused parameters

### CODE REVIEW

File: avaframe/in1Data/getInput.py [hunk 1]
```diff
@@ -65,59 +65,59 @@ def getDEMPath(avaDir):
...
-        demFile: pathlib path
-            path to dem file
+    Returns
+    --------
+    demFile: pathlib path
+        path to dem file
     """
 
-    if fileName == '':
-        demFilePath = getDEMPath(avaDir)
```
Comment (awirb) [inline avaframe/in1Data/getInput.py:107]: isn't this option lost in the new code? that if no fileName provided it looks for the dem in Inputs?

File: avaframe/in1Data/getInput.py [hunk 2]
```diff
@@ -145,59 +145,57 @@ def getInputData(avaDir, cfg):
...
                 raise FileNotFoundError(message)
             relFiles.append(relf)
-        log.debug('Release area file is specified to be: %s' % relFiles)
+        log.debug("Release area file is specified to be: %s" % relFiles)
     else:
-        releaseDir = 'REL'
-        relFiles = sorted(glob.glob(inputDir+os.sep + releaseDir+os.sep + '*.shp'))
-    log.info('Release area files are: %s' % relFiles)
+        releaseDir = "REL"
+        relFiles = sorted(glob.glob(inputDir + os.sep + releaseDir + os.sep + "*.shp"))
```
Comment (awirb) [inline avaframe/in1Data/getInput.py:171]: switch to pathlib instead of os?

File: avaframe/in1Data/getInput.py [hunk 3]
```diff
@@ -516,69 +509,65 @@ def fetchReleaseFile(inputSimFiles, releaseScenario, cfgSim, releaseList):
...
     # loop over all relFiles and compute information saved to dictionary of dataframes
-    relInfo = {}
-    relPath = pathlib.Path(avaDir, 'Outputs', 'com1DFA', 'releaseInfoFiles')
+    relPath = pathlib.Path(avaDir, "Outputs", "com1DFA", "releaseInfoFiles")
     fU.makeADir(relPath)
     relDFDict = {}
     # loop over all relFiles
     for relFile in relFiles:
         # for relF in relFiles:
-        releaseLine = {}
```
Comment (awirb) [inline avaframe/in1Data/getInput.py:576]: if it is not reinitialized as empty dict, doesnt it add/overwrite the keys?

**Review by awirb — CHANGES_REQUESTED** [review]

File: avaframe/in1Data/getInput.py (hunk 1 above)
Comment (fso42) [reply avaframe/in1Data/getInput.py:107]: demFilePath is never used, so this if makes no sense at all and only leads to an empty / undefined demFile being returned. Or am I missing something here?

File: avaframe/in1Data/getInput.py (hunk 3 above)
Comment (fso42) [reply avaframe/in1Data/getInput.py:576]: Mh, I'm pretty sure the keys are being overwritten, which is exactly what we want, right?

File: avaframe/in1Data/getInput.py (hunk 1 above)
Comment (fso42) [reply avaframe/in1Data/getInput.py:107]: ok, I'll include this then and make a pytest for it. Since we obviously missed it so far :-)

**Pull request merged**